root/entrypoint.sh

```diff
@@ -73,9 +73,16 @@ provisioner() {
 
     # Configure Postgres database
     if [ "$SYMFONY__ENV__DATABASE_DRIVER" = "pdo_pgsql" ] && [ "$POPULATE_DATABASE" = "True" ] && [ "$POSTGRES_PASSWORD" != "" ] ; then
+		if [ -z "$POSTGRES_DATABASE_NAME" ]; then
+			echo "WARN: POSTGRES_DATABASE_NAME is not set. POSTGRES_USER will be used as database name"
+		else
+			export PGDATABASE="${POSTGRES_DATABASE_NAME}"
+		fi
         export PGPASSWORD="${POSTGRES_PASSWORD}"
         DATABASE_EXISTS="$(psql -qAt -h "${SYMFONY__ENV__DATABASE_HOST}" -p "${SYMFONY__ENV__DATABASE_PORT}" -U "${POSTGRES_USER}" \
             -c "SELECT 1 FROM pg_catalog.pg_database WHERE datname = '${SYMFONY__ENV__DATABASE_NAME}';")"
+		TABLE_EXISTS="$(psql -qAt -h "${SYMFONY__ENV__DATABASE_HOST}" -p "${SYMFONY__ENV__DATABASE_PORT}" -U "${POSTGRES_USER}" \
+			-c "SELECT 1 FROM information_schema.tables WHERE table_name = 'wallabag_user';")"
```

Comment:
The table `wallabag_user` may have a different name on some installations as a prefix can be set using `$SYMFONY__ENV__DATABASE_TABLE_PREFIX`